File: app/src/main/java/com/androidacademy/msk/exerciseproject/network/model/NetworkImage.java
package com.androidacademy.msk.exerciseproject.network.model;

import android.support.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class NetworkImage {
    /**
     * url : https://static01.nyt.com/images/2018/10/24/nyregion/25CLINTON3-promo/25CLINTON3-promo-thumbStandard.jpg
     * height : 75
     * width : 75
     */

    @SerializedName("url")
    @Nullable
    private String url;

    @SerializedName("height")
    private int height;

    @SerializedName("width")
    private int width;

    @Nullable
    public String getUrl() {
        return url;
    }

    public void setUrl(@Nullable String url) {
        this.url = url;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }
}

File: app/src/main/java/com/androidacademy/msk/exerciseproject/data/network/model/NetworkImage.kt
package com.androidacademy.msk.exerciseproject.data.network.model

import com.google.gson.annotations.SerializedName


class NetworkImage {
    /**
     * url : https://static01.nyt.com/images/2018/10/24/nyregion/25CLINTON3-promo/25CLINTON3-promo-thumbStandard.jpg
     * height : 75
     * width : 75
     */

    @SerializedName("url")
    val url: String? = null

    @SerializedName("height")
    val height: Int = 0

    @SerializedName("width")
    val width: Int = 0

}